Gemfile.lock

```diff
@@ -169,4 +184,4 @@ RUBY VERSION
    ruby 2.6.8p205
 
 BUNDLED WITH
-   2.2.27
+   2.3.8
```

Comment:
why would this change for a test?

Reply:
I'll drop this change, it's not necessary.

Reply:
actually, it might be necessary, I had to add a few gems to facilitate the new test

Reply:
I added at least one new gem to facilitate the image test, and ran bundle to install the gems. This is the result.